### ISSUE TITLE
Possible Bug in aws-amplify/lib/Analytics/types/Analytics.d.ts

### ISSUE THREAD

**Issue by engharb** [aws/aws-amplify#675]
Bug

I ran ```$ npm install aws-amplify --save```. And when I start run ```$ ionic cordova run android``` I got the following exception:
```
typescript: node_modules/aws-amplify/lib/Analytics/types/Analytics.d.ts, line: 19 
            In ambient enum declarations member initializer must be constant expression. 

      L18:  export declare enum SessionState {
      L19:      START = "START",
      L20:      STOP = "STOP",
```
those two constants values must be numeric ones not string i.e. 
```
START = 1
STOP = 0
```
in order to solve problem from angular transpile.js ? Hope that will be fixed soon.

regards,

**Comment by richardzcode** [comment]
@engharb the problem is incompatibility between angular and typescript. Could you provide detailed steps so we can see if we could reproduce the issue?

Also, could you provide the output of these two command?

```
ng --version
ionic info
```

Thanks!

**Comment by engharb** [comment]
@richardzcode 
```
ionic info:
cli packages: (/usr/local/lib/node_modules)

    @ionic/cli-utils  : 1.19.1
    ionic (Ionic CLI) : 3.19.1

global packages:

    cordova (Cordova CLI) : 8.0.0 

local packages:

    @ionic/app-scripts : 3.1.8
    Cordova Platforms  : android 7.0.0
    Ionic Framework    : ionic-angular 3.3.0

System:

    Node  : v8.10.0
    npm   : 5.6.0 
    OS    : macOS High Sierra
    Xcode : Xcode 9.2 Build version 9C40b 

Environment Variables:

    ANDROID_HOME : not set

Misc:

    backend : pro
```

```
we are using angular 4 with Ionic 3
angular/core": "4.1.2",
```

**Comment by powerful23** [comment]
closing this issue as this is fixed in #670
